_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -33,109 +41,113 @@ class ChildInformationSection extends React.Component {
           </div>
         </div>
 
-        <div className="row">
-          <div className="small-12 columns">
-            <p>Child's Name</p>
-            <FullName
-                value={this.props.data.childFullName}
-                onUserInput={(update) => {this.props.onStateChange('childFullName', update);}}/>
-          </div>
-        </div>
-
-        <div className="row">
-          <div className="small-12 columns">
-            <p>Child's relationship to you (check one)</p>
-            <ErrorableSelect label="Child’s relationship to you"
-                options={childRelationships}
-                value={this.props.data.childRelation}
-                onValueChange={(update) => {this.props.onStateChange('childRelation', update);}}/>
+        {this.props.data.hasChildrenToReport === true &&
+          <div>
+            <div className="row">
+              <div className="small-12 columns">
+                <p>Child&rsquo;s Name</p>
```

Comment:
@akainic is correct that we don't need o specify this unless it is going into a database or something. But since this is text on the front-end and we are specifying the charset its not necessary.